.github/workflows/integration_test.yaml

```diff
@@ -21,15 +36,20 @@ jobs:
             value: "--num-units=3"
           - name: num_units=1, db_from_relation
             value: "--num-units=1"
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: canonical/operator-workflows/.github/workflows/integration_test_run.yaml@refactor-integration-test-job
```

Comment:
I suppose we'll wait for the branch `refactor-integration-test-job` to be merged before merging this one ?

Reply:
Yes